addons/account/models/product.py

```diff
@@ -139,6 +139,19 @@ def create(self, vals_list):
         products.invalidate_recordset(['taxes_id', 'supplier_taxes_id'])
         return products
 
+    def _get_gross_list_price(self, price):
```

Comment:
If those methods are only used in barcodelookup module, I think they could be defined directly in that module and not here :thinking:

Reply:
`barcodelookup` don't depend on `account` but needs to compute the price based on taxes. Do you prefer a bridge module instead ?

Reply:
Could have made sense, but not worth postponing something required for the OCD, leave it here then (and thanks for the explanation).